src/puppetlabs/trapperkeeper/authorization/acl.clj

```diff
@@ -11,7 +13,8 @@
 (def Extensions
   "Schema for representing SSL Extensions. Maps from a keyword shortname to a
   string value."
-  {schema/Keyword schema/Str})
+  {schema/Keyword schema/Str
+   (schema/optional-key :subjectAlternativeName) [schema/Str]})
```

Comment:
I haven't read far enough into this yet to know if this applies, but it seems like it would be good to allow the subj-alt-name value in the rule configuration to support both a list of strings or just a single string - given that we support the use of a single string in other parts of a rule.  If the value from the rule in the TK configuration is coerced into a vector before it is evaluated by this schema, my comment would be irrelevant.